get-involved/profile-badges/index.md

```diff
@@ -1,29 +1,35 @@
-- Assist the Make Accessible Team with the Weekly Meeting with administrative work such as posting an agenda or writing a follow-up for the meeting;
-- Update, suggest an edit, or draft a new documents in the Accessibility Team Handbook;
+- Facilitating a bug-scrub session.
+- Commit, Comment on, or report accessibility issues to the WordPress open-source project on [GitHub](https://github.com/wordpress) or on [Trac](https://core.trac.wordpress.org/).
+- Assist the Make Accessible Team with the Weekly Meeting with administrative work such as posting an agenda or writing a follow-up for the meeting.
+- Update, suggest an edit, or draft a new documents in the Accessibility Team Handbook.
 - Contribute to the accessibility of the WordPress.org network of sites, including OpenVerse, Learn.wordpress.org, wordpress.tv, or any of the other sites.
 
-If you believe you’ve earned the Contributor profile badge, post a request in the #accessibility Slack channel and somebody will follow up.
 
 ## Make Accessibility Team Profile Badge Criteria
 
-To receive a Team Profile Badge with the Make WordPress Accessible Team, you must actively contribute regularly or take a leadership role in the team.
+If you have served as a Accessibility Team Rep or have provided consistent substantial contributions to the Test Team, you are eligible for the Accessibility Team badge. These contributions include:
+
+- Participating in, contributing to, or moderating weekly team bugs scrubs.
+- Regularly commit, comment on, and report accessibility issues to the WordPress open-source project on [GitHub](https://github.com/wordpress) or on [Trac](https://core.trac.wordpress.org/).
+- Actively serve as Make WordPress Accessible Team Representative.
+- Attend and participate in Make WordPress Accessible Team meetings (currently bi-weekly).
+- Lead the Accessibility Table at a Contributor Day at a WordCamp.
+
+
+## Requesting a Accessibility Profile Badge
 
-This includes:
+If you meet the criteria above, please request a badge using the buttons below. Very important: Kindly include all the links to resources that demonstrate you have met the criteria mentioned above.
 
-- Participating in, contributing to, or moderating weekly team bugs scrubs;
-- Regularly commit, comment on, and report accessibility issues to the WordPress open-source project on [GitHub](https://github.com/wordpress) or on [Trac](https://core.trac.wordpress.org/);
-- Actively serve as Make WordPress Accessible Team Representative;
-- Attend and participate in Make WordPress Accessible Team meetings (currently bi-weekly);
-- Lead the Accessibility Table at a Contributor Day at a WordCamp
+- [Contributor Badge](https://profiles.wordpress.org/associations/accessibility-contributor/)
+- [Team Badge](https://profiles.wordpress.org/associations/accessibility-team/)
```

Comment:
Remove the link to the Team Badge request. Team badges will be by invitation only.